### PR TITLE
adds pie chart view /contributions with dummy data

### PR DESCRIPTION
**Pull request by cmacaulay**
👍 
- there's a pie chart! this means the library that is being brought in is working
👎 
- no real data yet! stay tuned
screen shot 2017-05-25 at 6 21 51 pm

### CODE REVIEW

File: public_officials/templates/public-officials/contributions.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+
+<body>
+    <h1>Fruits vs Calories</h1>
+    {% include_container chartcontainer 400 600 %}
+</body>
```
Comment (meyerhoferc) [inline public_officials/templates/public-officials/contributions.html]: 👍

File: public_officials/templates/public-officials/detail.html [hunk 2]
```diff
@@ -157,6 +161,17 @@
...
+        {% load nvd3_tags %}
+        <div class="container-fluid">
+          <div class="row-well">
+            <div class="data-dashboard">
+              <h1>Fruits vs Calories</h1>
+              {% include_container chartcontainer 400 600 %}
+            </div>
+          </div>
+        </div>
+      </div>
```
Comment (meyerhoferc) [inline public_officials/templates/public-officials/detail.html]: awesome steps here to get started

File: public_officials/views.py [hunk 3]
```diff
@@ -5,6 +5,7 @@
 from django.core.cache import cache
 from public_officials.services import *
 from public_officials.models import *
+from nvd3 import pieChart
```
Comment (meyerhoferc) [inline public_officials/views.py]: 👍 do we need to pip install this? @cmacaulay

File: public_officials/views.py [hunk 4]
```diff
@@ -110,6 +111,25 @@ def bills_index(request):
...
+        'chartdata': chartdata,
+        'chartcontainer': chartcontainer,
+        'extra': {
+            'x_is_date': False,
+            'x_axis_format': '',
+            'tag_script_js': True,
+            'jquery_on_ready': False,
+        }
+    }
+    return render(request, 'public-officials/contributions.html', data)
```
Comment (meyerhoferc) [inline public_officials/views.py]: great start with data prep.

File: will_legislate_for_money/settings.py [hunk 5]
```diff
@@ -39,6 +39,8 @@
     'django.contrib.messages',
     'django.contrib.staticfiles',
     'public_officials',
+    'djangobower',
+    'django_nvd3',
```
Comment (meyerhoferc) [inline will_legislate_for_money/settings.py]: 👍

File: will_legislate_for_money/urls.py [hunk 6]
```diff
@@ -26,6 +26,7 @@
     url(r'^senators/$', public_official_views.senator_index, name="senator_index"),
     url(r'^representatives/$', public_official_views.representative_index, name="representative_index"),
     url(r'^recent-bills/$', public_official_views.bills_index, name="bills_index"),
+    url(r'^contributions/$', public_official_views.contributions, name="contributions"),
```
Comment (meyerhoferc) [inline will_legislate_for_money/urls.py]: 🎉

File: public_officials/templates/public-officials/contributions.html (hunk 1 above)
Comment (JonathanSR) [inline public_officials/templates/public-officials/contributions.html]: Charts for the win!

**Comment by JonathanSR** [comment]
Everything looks good to me.
Good job @cmacaulay

**Pull request merged**